mita/openstack.py

```diff
@@ -111,80 +100,84 @@ def get_driver(self, **kw):
             ex_tenant_domain_id=self.tenant_domain_id,
         )
 
-    def get_driver_v1(self):
-        """
-        Get Apache Libcloud OpenStack driver for nova version 1.1.
+    def _get_image_by_name(self) -> NodeImage:
+        """Return the glance image reference."""
+        logger.info("Gathering %s image details.", self.image_name)
 
-        https://libcloud.readthedocs.io/en/latest/compute/drivers/
-        openstack.html#compute-1-0-api-version-older-installations
+        url = f"/v2/images?name={self.image_name}"
+        _object = self.driver.image_connection.request(url).object
+        images = self.driver._to_images(_object, ex_only_active=False)
```

Comment:
`_to_images()` -> `_to_image()` for simplicity. We can also drop `ex_only_active` as well.

Reply:
Whoops, I'm sorry, you're right this returns multiple images!

Reply:
But we do want to filter by active images, right? `ex_only_active=True`?

Reply:
It should be `True`... it is a collateral from my experiments. Thanks... will fix it

Reply:
Actually... it is a problem with libcloud... they look for upper case instead of lower case which prevents us from picking up the correct image.